src/utils.h

```diff
@@ -177,17 +177,22 @@ static inline int get_index_hash(void *p, int max, int struct_size, uint32_t key
 #define LOGL(level, a, ...)                             \
 	{                                                   \
 		if ((level)&opts.log)                           \
-			_log(__FILE__, __LINE__, a, ##__VA_ARGS__); \
+			_log(__FILE__, __LINE__, level, a, ##__VA_ARGS__); \
+	}
+#define LOG(a, ...)                                     \
+	{                                                   \
+			_log(__FILE__, __LINE__, DEFAULT_LOG, a, ##__VA_ARGS__); \
+	}
+#define LOGG(level, a, ...)                                     \
```

Comment:
There is no need for LOGG, there is LOGL doing the same thing

Reply:
This is necessary because `LOGL(...)` prints only if the module has activated the log functionality, and the `LOGG(...)` prints every time. This modification of two different functions is necessary to support the _inconditional_ log of a module even if the log of the module is not enabled. Take note that `LOG(...)` and `LOG0(...)` do not print the name of the module.

Reply:
But that means that you are printing the message regardless if http was enabled or no, right ?  If you want to print it in the http log level (or rtsp) feel free to use LOGL.